apps/ledger-live-desktop/tests/specs/speculos/README.md

```diff
@@ -82,5 +82,9 @@ pnpm desktop build:testing
 ## Run tests
 
 ```
-pnpm desktop test:playwright:speculos specs/<testFolder>/<testName>
+pnpm desktop test:playwright:speculos specs/speculos/<testName>
+```
+or
+```
+pnpm desktop test:playwright:speculos ./<testName> --config=tests/playwright.config.ts
```

Comment:
I don't understand what is achieved for testers through this second command. 
The `--config=tests/playwright.config.ts` part seems redondant.

Being picky but I think `<testName>` should be `<testFileName>`

IMO, there is no need for the relative path `specs/speculos/` as it is given by the speculos project. 
We could summarize both commands with just :
`pnpm desktop test:playwright:speculos <testFileName>`

Reply:
- I added the second command because the first one wasn't working for @bharamboure-ledger.
- Going to change the path :)